.gitignore

```diff
@@ -2,3 +2,5 @@ trainingset/*
 training_data/*
 tmp/*
 .ipynb_checkpoints/*
+*.mp4
+*.mkv
```

Comment:
This shows a warning saying it wants the file terminating with a blank line

Reply:
Shuld be fixed with commit "fix last blank line"